src/app/stack/card-details/report-information/report-information.component.html

```diff
@@ -11,7 +11,7 @@
             <b>Register</b> for a <b>free</b> Snyk account to find out more, including whether any of them have a publicly known exploit.
         </span>
         <span class="col-sm-2">
-            <a [href]="this.generateUrl.registrationURL" target="_blank" title="command/ctrl + click"
+            <a [href]="this.generateUrl.registrationURL" target="_blank"
                 (click)="this.handleUrl.open($event, this.generateUrl.registrationURL)"><button class="ripple">Sign up for
```

Comment:
Shouldn't we do preventDefault on the click event?

Reply:
Ya correct, I forgot to do that. Added it now.